src/event/views/CreateEventPage.vue

```diff
@@ -1,162 +1,186 @@
 <template>
-  <form class="form">
-    <!-- Left Components -->
-    <!-- Date Card -->
-    <section class="date card">
-      <h5 class="heading">Pick the date</h5>
-      <!-- EventDateSelectors component -->
-      <DatePicker
-        v-if="isDatePickerEvent"
-        v-model:startTime="startTime"
-        v-model:endTime="endTime"
-        class="date-picker"
-      />
-      <DayPicker
-        v-if="!isDatePickerEvent"
-        v-model:startTime="startTime"
-        v-model:endTime="endTime"
-        class="day-picker"
-      />
-      <AppToggleExternalText
-        class="date-toggle"
-        toggle-left-text="Single"
-        toggle-right-text="Recurring"
-        :default-state="false"
-        @toggled="toggleEventType"
-      />
-      <AppToggleInternalText
-        style="display: none"
-        left-text="My Availability"
-        right-text="Group Availability"
-        :checked="false"
-        @update="toggleEventType()"
-      />
-    </section>
-
-    <!-- Right Components -->
-    <!-- Time  Card -->
-    <section class="time card">
-      <h5 class="heading">Pick the time</h5>
-
-      <!-- EventTimePicker component -->
-      <div class="time-picker">
-        Between
-        <TimePicker v-model="startTime" />
-        and
-        <TimePicker v-model="endTime" />
+  <div class="container">
```

Comment:
Hmm is this the old CreateEventPage by any chance? @tnwin can you clarify?

Reply:
Old according to #67 for sure.